R/construct_label.R

```diff
@@ -1,12 +1,14 @@
-#' Build labels with significances represented in bold or as raised 'a', and, if wanted, standard errors in brackets. Main usage for plotting tables and brace labels.
+#' Build labels with significances represented in bold or as raised 'a', and, if wanted, standard errors in brackets. Main usage for plotting tables and brace labels. #with significant values or significant labels? #whats a 'raised a' - 'superscripted a'?
 #'
 #' @inheritParams plot_lineplot
 #' @param dat Dataframe with the columns that should be merged into labels.
-#' @param label_sig_high_extra_column Logical indicating whether the superscript for significances should be added directly into the label (necessary for lineplots), or should be written into an extra column with the ending '_sig_superscript' (necessary for tables).
+#' @param label_sig_high_extra_column Logical indicating whether the superscript for significances should be added directly into the label (necessary for lineplots), or should be written into an extra column with the ending '_sig_superscript' (necessary for tables). #which is which?
 #' @param new_name Character string for the new column that is added to `dat`. Defaults to `'label'`.
 #' @param round_est Rounding of label_est.
 #' @param round_se Rounding of label_se.
 #'
+#'          # are parameters missing? (see most 'label' parameters in the function below)
```

Comment:
No, parameters that are also part of other functions can be inherited by `@inheritParams` (see first parameter). This makes sense, so the documentation doesn't get duplicated.